### PR TITLE
Add dynamic syntax highlighting

### PR DESCRIPTION
**Pull request by AlexExcellence**
- readded the dynamic inline feedback button that got lost during merge
- added dynamic syntax highlighting for swift and java based on the file extension of the currently selected file

### CODE REVIEW

File: Themis/Views/Assessment/CodeEditor/CodeView.swift [hunk 1]
```diff
@@ -12,14 +13,36 @@ struct CodeView: UIViewControllerRepresentable {
...
+            case .swift:
+                viewController.textView.setLanguageMode(TreeSitterLanguageMode(language: .swift), completion: { _ in
+                    viewController.textView.text = code })
+            case .java:
+                viewController.textView.setLanguageMode(TreeSitterLanguageMode(language: .java), completion: { _ in
+                    viewController.textView.text = code })
+            case .other:
+                viewController.textView.setLanguageMode(PlainTextLanguageMode(), completion: { _ in
+                    viewController.textView.text = code })
+            }
```
Comment (tomrudnick) [inline Themis/Views/Assessment/CodeEditor/CodeView.swift]: Maybe this could be in a method of the ViewController so that we don't have duplicated code

**Review by tomrudnick — APPROVED** [review]
Looks good

File: Themis/Views/Assessment/CodeEditor/CodeView.swift (hunk 1 above)
Comment (AlexExcellence) [reply Themis/Views/Assessment/CodeEditor/CodeView.swift]: Moved it in the ViewModel 👍

**Review by crustypluto19 — APPROVED** [review]
Looks good to me! :)

**Review by botrufus — APPROVED** [review]

**Pull request merged**